bin/is-website-vulnerable.js

```diff
@@ -4,9 +4,13 @@
 
 const debug = require('debug')('is-website-vulnerable')
 const { Audit, RenderConsole } = require('../index')
+const Utils = require('../src/Utils')
```

Comment:
@SoldierCorp you could just add 'Utils' is the 3rd element to destructure in line 6 instead of requiring that again, since index.js is already getting used. Right?

Reply:
You right, my bad.